fe1-web/src/features/social/components/ChirpCard.tsx

```diff
@@ -22,70 +23,36 @@ import { makeReactionsList } from '../reducer';
  */
 
 const styles = StyleSheet.create({
-  container: {
-    borderColor: gray,
-    borderTopWidth: 0,
-    borderWidth: 1,
-    flexDirection: 'column',
-    padding: 10,
-    width: 600,
+  profileIcon: {
+    alignSelf: 'flex-start',
   } as ViewStyle,
-  innerContainer: {
-    flexDirection: 'row',
-  } as ViewStyle,
-  leftView: {
-    width: 60,
-  } as ViewStyle,
-  rightView: {
-    display: 'flex',
-    flexDirection: 'column',
-  } as ViewStyle,
-  senderText: {
-    fontSize: 18,
-    fontWeight: '600',
-  } as TextStyle,
-  senderView: {
-    fontSize: 18,
-    marginTop: 7,
-  } as ViewStyle,
-  chirpText: {
-    fontSize: 18,
-    paddingBottom: 20,
-    paddingTop: 10,
-    width: 520,
-  } as TextStyle,
-  deletedChirpText: {
-    fontSize: 18,
```

Comment:
The typography related styles could be replaced by `style={Typography.xxx}` and the fixed widths were removed in order to make the application always fill the full width. Otherwise it is not usable on certain (especially mobile) devices. A solution to make it better to use on desktop would be something like a bootstrap container (https://getbootstrap.com/docs/5.0/layout/containers/) wrapped around the whole application.